tests/base-notebook/test_packages.py

```diff
@@ -47,30 +47,31 @@
 # Mapping between package and module name
 PACKAGE_MAPPING = {
     # Python
-    "matplotlib-base": "matplotlib",
     "beautifulsoup4": "bs4",
-    "scikit-learn": "sklearn",
+    "matplotlib-base": "matplotlib",
+    "pytables": "tables",
     "scikit-image": "skimage",
+    "scikit-learn": "sklearn",
     "spylon-kernel": "spylon_kernel",
-    "pytables": "tables",
     # R
     "randomforest": "randomForest",
-    "rsqlite": "DBI",
     "rcurl": "RCurl",
     "rodbc": "RODBC",
+    "rsqlite": "DBI",
 }
 
 # List of packages that cannot be tested in a standard way
 EXCLUDED_PACKAGES = [
-    "python",
-    "hdf5",
+    "bzip2",
+    "ca-certificates",
     "conda-forge::blas[build=openblas]",
+    "hdf5",
+    "mamba[version='<0.18']",
```

Comment:
I sorted these 2 lists, but also added this line - it's a workaround for testing under arm.
As soon as we remove this workaround for arm, this line can be deleted.

Reply:
> it's a workaround for testing under arm.
As soon as we remove this workaround for arm, this line can be deleted.

That sounds like an excellent inline comment!

```suggestion
    # FIXME: When we remove a workaround for arm regarding mamba, we can 
    #        test installation of mamba as well and remove this exception.
    #        See ???
    "mamba[version='<0.18']",
```

I like to explain a bit that something can be removed in the future, and link out to an issue where it is tracked, so something like the comment above but with a functional issue to understand if its time to remove it.

Reply:
I agree. Done 👍